aws/components/cdn/setup.ftl

```diff
@@ -14,10 +14,10 @@
     [#local cfName              = resources["cf"].Name]
 
     [#local wafPresent          = isPresent(solution.WAF) ]
-    [#local wafVersion          = (solution.WAF.Version)!"V1" ]
+    [#local wafVersion          = (solution.WAF.Version)!"v1" ]
```

Comment:
This is inconsistent with the defaults in the engine which sets v2 as the default. Given it is being defaulted via the engine configuration, do we need to catch a missing value at this point?

I wonder if initially it would be better to leave the default as v1 so existing deployments aren't affected and new ones can pick v2 from the get go. At some future point, we then change the default to v2 once we are confident the bulk of installations have transitioned.

Reply:
Because solution.WAF may be absent (refer L16), I added that to make sure that it was always a text string even when absent to simplify the code. While I may have made an error in working through it, I did hit issues when WAF was not expected to be set

I had it that way initially for that reason. Happy to revert